topics/ui/index.md

```diff
@@ -0,0 +1,10 @@
+---
+aliases: user-interface
+created_by: Glenda Schroeder
+display_name: User interface
+released: '1985'
+short_description: A UI is what users use to control a program through a graphical interface
```

Comment:
The qualification of "through a graphical interface" seems misleading here. That should be the territory of `GUI`. Wikipedia has a nice succinct description we could use here:
```suggestion
short_description: A user interface (UI) is the space where interactions between humans and machines occur.
```